adb_android/adb_android.py

```diff
@@ -174,6 +174,13 @@ def _convert_opts(opts):
     """
     return ' '.join(opts)
 
+def adb_help():
+    """
+    Prints the adb help
+    """
+    helpC = v.ADB_COMMAND_HELP
```

Comment:
Could you think about a more describing name for this variable?

Reply:
Okay fine, I'll do it!